src/instructlab/model/backends/backends.py

```diff
@@ -219,31 +219,34 @@ def ensure_server(
         # pylint: enable=duplicate-code
     except ClientException:
         port = free_tcp_ipv4_port(host)
-        logger.debug("Using %i", port)
+        logger.debug(f"Using available port {port} for temporary model serving.")
 
         host_port = f"{host}:{port}"
         temp_api_base = get_api_base(host_port)
-        logger.debug(f"Starting a temporary server at {temp_api_base}...")
         llama_cpp_server_process = None
         vllm_server_process = None
 
         if backend == VLLM:
             # TODO: resolve how the hostname is getting passed around the class and this function
             vllm_server_process = server_process_func(port)
+            logger.info(f"Starting a temporary vLLM server at {temp_api_base}")
             count = 0
             # TODO should this be configurable?
-            vllm_startup_timeout = 300
+            vllm_startup_timeout = 60
             while count < vllm_startup_timeout:
-                sleep(1)
```

Comment:
@JamesKunstle what's the logic in removing the sleep and reducing the timeout to 60? How long could you expect this loop to go for?

I've had times where I've noticed vLLM not be ready for almost 2 minutes when I run things locally. I set the default of 300 to be on the safe side.